index.html

```diff
@@ -2252,14 +2252,36 @@
       <h3>Operation</h3>
       <p>A <code><a>RTCDtlsTransport</a></code> instance is constructed using an
       <code><a>RTCIceTransport</a></code> and a sequence of
-      <code><a>RTCCertificate</a></code> objects. If <code>certificates</code> is
-      non-empty, check that the <code>expires</code> attribute of each
-      <code><a>RTCCertificate</a></code> object is in the future. If a certificate has
-      expired, throw an InvalidParameter exception; otherwise, store the
-      certificates.</p>
+      <code><a>RTCCertificate</a></code> objects.
+      An <code><a>RTCDtlsTransport</a></code> object in the <code>closed</code> or
+      <code>failed</code> states can be garbage-collected when it is no longer
+      referenced.</p>
+      <p>Since the Datagram
+      Transport Layer Security (DTLS) negotiation occurs between transport endpoints
+      determined via ICE, implementations of this specification <em class="rfc2119"
+      title="MUST">MUST</em> support multiplexing of STUN, TURN, DTLS and RTP and/or
+      RTCP. This multiplexing, originally described in [[!RFC5764]] Section 5.1.2, is
+      updated in [[!RFC7983]].</p>
+      <p>On construction, the [[role]] internal slot is set to
```

Comment:
It may not be entirely clear why there's an internal slot for `role`. Maybe the *role internal slot that represents the actual DTLS role*?